tools/cldr-apps/js/src/views/ForumButton.vue

```diff
@@ -0,0 +1,104 @@
+<script setup>
```

Comment:
"script setup" appears to be the latest recommended syntax, and so is putting script, template, style in that order

See:

https://vuejs.org/api/sfc-script-setup
https://vuejs.org/style-guide/rules-recommended#single-file-component-top-level-element-order

Reply:
it's very handy, has some caveats.